examples/terraform-aws-s3-example/locals.tf

```diff
@@ -0,0 +1,10 @@
+# ---------------------------------------------------------------------------------------------------------------------
```

Comment:
While I agree that this is good practice, can we have this at the bottom of `main.tf` so that it is consistent with the other examples?